src/Components/Counter/Counter.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+const title = 'Counter';
+
+export class Counter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: props.initialValue || 0
+        }
+    }
+
+    operateCounter(action) {
+        this.setState((prevState) => ({ counter: action === 'increase' ? prevState.counter + 1 : prevState.counter - 1 }));
```

Comment:
Ideally, 'increase' and 'decrease' actions can be like constants. (Action.Increase, or something like it)